src/modules/bootloader/main.py

```diff
@@ -244,9 +244,10 @@ def create_loader(loader_path, installation_root_path):
     :param installation_root_path: The path to the root of the target installation
     """
 
+    #Obsolete since default was changed to @saved from machine-id
```

Comment:
Add a space after `#`

Reply:
Changed to using a multi line comment. Thanks